src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,18 @@
 package core.basesyntax;
 
+import models.Ball;
+import models.Lottery;
+
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[3];
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = lottery.getRandomBall();
+        }
+        for (int i = 0; i < balls.length; i++) {
+            System.out.println(balls[i].toString());
```

Comment:
toString will be called implicitly
```suggestion
            System.out.println(balls[i]);
```

Reply:
++